packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -1,20 +1,13 @@
 /* External Imports */
 import { Promise as bPromise } from 'bluebird'
```

Comment:
What do you think of removing `bluebird` as a dependency in the future? Concurrent async calls are easy with `Promise.all`

Reply:
I had the same thought when I read that code before, here's a post providing good reasoning on "why bluebird" https://blog.kuzzle.io/bluebird-native-async_await-javascript-promises-performances-2020